x/packages/types/proposal.go

```diff
@@ -0,0 +1,59 @@
+package types
+
+import (
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	epochstoragetypes "github.com/lavanet/lava/x/epochstorage/types"
+)
+
+// Function to verify a package object fields
+func checkPackagesProposal(packageToCheck Package) error {
```

Comment:
lets call this ValidatePackage and make it a function of the package struct

Reply:
done